src/bench/bench_bitcoin.cpp

```diff
@@ -60,6 +61,17 @@ static uint8_t parsePriorityLevel(const std::string& str) {
     return levels;
 }
 
+// Parses test setup related arguments
+static std::vector<std::string> parseTestSetupArgs(const ArgsManager& argsman) {
```

Comment:
> I don't feel strong about the capitalization neither but it looks incorrect to introduce it now when all other functions aren't using it.

(`SetupBenchArgs` is capitalized, `main` must be non-capitalized, so that only leaves `parseAsymptote` to change beyond the function under discussion).